HW6/SoKKoS/HW6_3.py

```diff
@@ -0,0 +1,16 @@
+def func_dict(input_txt):
```

Comment:
another variant:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result